.gitattributes

```diff
@@ -70,6 +70,8 @@ tools/mantis2gh_stripped.csv typo.missing-header
 # ignore auto-generated .depend files
 .depend                  typo.prune
 /.depend.menhir          typo.prune
+# ignore local opam switch
+/_opam                   typo.prune
```

Comment:
This becomes unnecessary if/when #10656 is merged

Reply:
(which it now is!)

Reply:
I update my commit (and I know the difference between moving a branch and rebasing for the first time)